web/beacon-app/src/features/teams/components/RegisterNewUser/NewInviteRegistrationForm.tsx

```diff
@@ -0,0 +1,163 @@
+  privacy_agreement: false,
+} satisfies NewInvitedUserAccount;
+
+type RegistrationFormProps = {
+  onSubmit: (values: NewInvitedUserAccount, helpers: FormikHelpers<NewInvitedUserAccount>) => void;
+};
+
+function NewInviteRegistrationForm({ onSubmit }: RegistrationFormProps) {
+  const formik = useFormik<NewInvitedUserAccount>({
+    initialValues,
+    onSubmit,
+    validationSchema: validationSchema,
+  });
+  const { touched, errors, values, getFieldProps, setFieldValue, isSubmitting } = formik;
+
+  const [isFocused, { onBlur, onFocus }] = useFocus();
+  // eslint-disable-next-line unused-imports/no-unused-vars
+  const [isPasswordMatchOpen, setIsPasswordMatchOpen] = useState<boolean | undefined>(
+    !!values.password
+  );
+  const isMobile = useMedia('(max-width: 860px)');
+
+  const handlePasswordMatch = (_result: boolean) => {
+    // console.log('result', result)
+  };
+
+  useEffect(() => {
+    setIsPasswordMatchOpen(!!values.password);
+    setTimeout(() => {
+      setIsPasswordMatchOpen(undefined);
+    }, 10000);
+  }, [values.password]);
+
+  return (
+    <FormikProvider value={formik}>
+      <Form>
+        <div className="mb-1 space-y-2">
+          <TextField
+            label={`Name (required)`}
+            placeholder="Holly Golightly"
+            data-testid="name"
+            fullWidth
+            errorMessage={touched.name && errors.name}
+            {...getFieldProps('name')}
+          />
+          <TextField
+            label={`Email address (required)`}
+            placeholder="holly@golight.ly"
+            fullWidth
+            data-testid="email"
+            errorMessage={touched.email && errors.email}
+            {...getFieldProps('email')}
+          />
+          <div className="relative">
+            <RadixTooltip.Provider>
+              <RadixTooltip.Root open={isFocused}>
+                <RadixTooltip.Trigger asChild>
+                  <div>
+                    <PasswordField
+                      placeholder={`Password`}
+                      data-testid="password"
+                      errorMessage={touched.password && errors.password}
+                      fullWidth
+                      {...getFieldProps('password')}
+                      onFocus={onFocus}
+                      onBlur={onBlur}
+                    />
+                  </div>
+                </RadixTooltip.Trigger>
+                <RadixTooltip.Portal>
+                  <RadixTooltip.Content
+                    className="select-none rounded-[4px] bg-white px-[15px] py-[10px] text-xs text-[15px] leading-none shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] will-change-[transform,opacity] data-[state=delayed-open]:data-[side=top]:animate-slideDownAndFade data-[state=delayed-open]:data-[side=right]:animate-slideLeftAndFade data-[state=delayed-open]:data-[side=left]:animate-slideRightAndFade data-[state=delayed-open]:data-[side=bottom]:animate-slideUpAndFade"
```

Comment:
I think those classnames already exist in the `tailwind.config.ts`

Reply:
I saw the class names that come after `animate` in the Tailwind config file. I wanted the style to agree with the `Registration Form` we have now, so I used the same class names I saw in that file.